src/interval.jl

```diff
@@ -120,6 +120,8 @@ Base.first(interval::Interval) = interval.first
 Base.last(interval::Interval) = interval.last
 isclosed(interval::AbstractInterval{T,L,R}) where {T,L,R} = L === Closed && R === Closed
 Base.isopen(interval::AbstractInterval{T,L,R}) where {T,L,R} = L === Open && R === Open
+containsfirst(interval::AbstractInterval{T,L,R}) where {T,L,R} = L === Closed 
+containslast(interval::AbstractInterval{T,L,R}) where {T,L,R} = R === Closed 
```

Comment:
I think this functionality is useful but the name needs more consideration. There may be a rename of first/last in the near future: https://github.com/invenia/Intervals.jl/issues/90